canonical-web-and-design/ubuntu-com-security-api
# Fix Swagger static files

## Done

Change url to static files.

The package did not allow to override the url of the static files, so I manually extended the `FlaskApiSpec` class to override the  `add_swagger_routes` function. That forced me to copy-paste the static files to our repo.

Plus some small url changes for /docs and /spec.json.

## QA
- Clone repo & fetch changes
- docker-compose up -d
- `dotrun`
- Browse to http://0.0.0.0:8030/security/api/docs you should see the documentation proving that everything works. 
The endpoings cannot be tested unless there is a local database.

## Issue

Fixes #6
